kura/org.eclipse.kura.ai.triton.server/OSGI-INF/metatype/org.eclipse.kura.ai.triton.server.TritonServerContainerService.xml

```diff
@@ -158,6 +158,51 @@
             default="4194304"
             min="1">
         </AD>
+        
+        <AD id="enable.metrics"
+            name="Enable Triton Server Metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="true"
+            description="Enable the Triton Server Metrics feature. This property enables all the default metrics..">
+        </AD>  
+        
+        <AD id="enable.gpu.metrics"
+            name="Enable Triton Server GPU Metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="true"
+            description="Enable the Triton Server GPU Metrics. It requires that the Enable Triton Server Metrics property is set to true.">
+        </AD> 
+        
+        <AD id="enable.cpu.metrics"
+            name="Enable Triton Server CPU Metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="true"
+            description="Enable the Triton Server CPU Metrics. It requires that the Enable Triton Server Metrics property is set to true.">
+        </AD>      
+        
```

Comment:
🤔  I'm worried about the exploding number of configuration parameters. Can we take some decisions for our users? What if `enable.metrics` enable both `cpu` and `gpu`? Is that feasible?

```suggestion
        <AD id="enable.metrics"
            name="Enable Triton Server Metrics"
            type="Boolean"
            cardinality="0"
            required="true"
            default="true"
            description="Enable the Triton Server Metrics feature. This property enables all the default metrics..">
        </AD>  
        
```

Reply:
I'm worried too. Currently we have the following parameters:

- default metrics
- GPU metrics
- CPU metrics
- metrics config

I'd preserve the metrics enable and config, setting GPU and CPU enabled by default.